src/oaipmh/client.py

```diff
@@ -338,9 +338,11 @@ def buildHeader(header_node, namespaces):
 def ResumptionListGenerator(firstBatch, nextBatch):
     result, token = firstBatch()
     while 1:
+        itemFound = False
```

Comment:
`item_found` for PEP8 compliant variables please? Then I'd call this mergeable unless it has further consequences I don't understand @jascoul

Reply:
Looking at the rest of the file, it's camelCase everywhere… but sure!
I would not merge this until Travis runs on PRs, also.

Reply:
good points. but yeah, i think at least that can be manually triggered?

Reply:
sure, I technically have the rights to do it…